src/base/static/components/input-form/input-form-category-selector.js

```diff
@@ -36,17 +52,26 @@ class InputFormCategorySelector extends Component {
               }}
               onExpandCategories={() => this.setState({ isCollapsed: false })}
             />
-          );
+          ) : null;
```

Comment:
nit: I believe we can replace this ternary with `&&`, since we are returning `null` anyway